src/nvim/move.c

```diff
@@ -930,11 +943,7 @@ void curs_columns(win_T *wp, int may_scroll)
     redraw_later(wp, NOT_VALID);
   }
 
-  // Redraw when w_virtcol changes and 'cursorcolumn' is set
-  if (wp->w_p_cuc && (wp->w_valid & VALID_VIRTCOL) == 0
-      && !pum_visible()) {
-    redraw_later(wp, SOME_VALID);
-  }
+  redraw_for_cursorcolumn(curwin);
```

Comment:
Should this not be `redraw_for_cursorcolumn(wp);`?

Reply:
Oops

Reply:
I think I included the change in one of the smoothscroll patches.